internal/executor/instance/persistentworker/isolation/none/none.go

```diff
@@ -83,3 +84,7 @@ func (pwi *PersistentWorkerInstance) WorkingDirectory(projectDir string, dirtyMo
 
 	return pwi.tempDir
 }
+
+func (pwi *PersistentWorkerInstance) Close() error {
+	return pwi.cleanup()
```

Comment:
Do you think it will make sense to try first use `$TMP/cirrus-build` in here https://github.com/cirruslabs/cirrus-cli/blob/865c22b71f41669d17b11cfff6227638b922bf4c/internal/executor/instance/persistentworker/isolation/none/none.go#L28 before generating a randomone? Absolute paths in logs will be prettier and most likely consistent.

Reply:
Thanks, almost forgot about that, see https://github.com/cirruslabs/cirrus-cli/pull/356/commits/f581e5194f8ca9ba8514eba63d33bfaab3fdc5ce.